tools/codegen/api/types.py

```diff
@@ -641,9 +637,9 @@ def inner_call(self) -> str:
     def from_func(f: NativeFunction, *, functional_op: NativeFunction, is_reverse: bool) -> 'FunctionalizationLambda':
         # Some assertions: lambdas are only used for view ops
         assert f.is_view_op
-        assert functional_op.is_view_op
+        assert not functional_op.is_view_op
```

Comment:
lol

Reply:
(i felt the same way lol)

Reply:
Context: before this PR, we have view_inplace kernels that would redispatch to the view kernel (`transpose_` to `transpose`).  So the "functional_op" here was `transpose`, and both were view ops.

Now, `f` will be either a view or view_inplace op, and `functional_op` will always be the view_copy op.